lib/airbrake-ruby.rb

```diff
@@ -286,7 +286,9 @@ def call_notifier(notifier, method, *args, &block)
   end
 end
 
-# Notify of unhandled exceptions, if there were any.
+# Notify of unhandled exceptions, if there were any, but ignore SystemExit.
 at_exit do
-  Airbrake.notify_sync($ERROR_INFO) if $ERROR_INFO
+  if $ERROR_INFO && $ERROR_INFO.class != SystemExit
```

Comment:
Okay, thanks for looking. I'm that case, maybe you can improve the comment to explain why we're ignoring `SystemExit`?